environments/contrail/install_vrouter_kmod.yaml

```diff
@@ -303,7 +304,7 @@ resources:
                   ip route add default via $def_gw
                 fi
               fi
-              if [[ ${dpdk_driver} == "vfio-pci" || ${sriov_on} == 'true' ]]; then
+              if [[ ${dpdk_driver} == "vfio-pci" || ${sriov_on} == 'True' ]]; then
                 sed 's/^\(GRUB_CMDLINE_LINUX=".*\)"/\1 $KERNEL_ARGS intel_iommu=on iommu=pt"/g' -i /etc/default/grub ;
```

Comment:
for vfio-pci should be : intel_iommu=on iommu=pt
for sriov:                         intel_iommu=on

so, it is needed to to chek it here additionally